include/uv.h

```diff
@@ -555,7 +555,10 @@ UV_EXTERN int uv_tcp_open(uv_tcp_t* handle, uv_os_sock_t sock);
 UV_EXTERN int uv_tcp_nodelay(uv_tcp_t* handle, int enable);
 UV_EXTERN int uv_tcp_keepalive(uv_tcp_t* handle,
                                int enable,
-                               unsigned int delay);
+                               unsigned int delay, 
+                               unsigned int interval, 
+                               unsigned int count);
```

Comment:
The v1.x branch is API and ABI stable. That means you can't change function signatures (or the sizes of structs, etc.)

Introducing a new `uv_tcp_keepalive_ex()` function is okay, that can land in a minor release.